src/engines/fx/lpc/lpc.hpp

```diff
@@ -1,9 +1,11 @@
 #pragma once
-
+#include <vector>
+#include "core/audio/processor.hpp"
 #include "core/engine/engine.hpp"
+#include "Gamma/Noise.h"
 
 #define MAXIMUM_LPC_ORDER 50
-
+#define MAXIMUM_BLOCK_SIZE 1024
```

Comment:
Get those macros out of here! 😉 
You want a `static constexpr int max_order = 50` member of the LPC engine.